cd/lib/jwt.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+#
+# JWT Encoder Bash Script; adapted from https://www.willhaley.com/blog/generate-jwt-with-bash/
+#
+
+usage() { echo "Usage: $0 -k </path/to/github/private/key> -i <github_app_id>" 1>&2; exit 1; }
+
+JWT_SIGNING_ALGO="RS256"
+
+while getopts ":k:i:" arg; do
+  case "${arg}" in
+    k)
+      GITHUB_PRIVATE_KEY_PATH=${OPTARG}
+      ;;
+    i)
+      GITHUB_APP_ID=${OPTARG}
+      ;;
+    *)
+      usage
+      ;;
+  esac
+done
+shift $((OPTIND-1))
+
+if [ -z "${GITHUB_APP_ID}" ] || [ -z "${GITHUB_PRIVATE_KEY_PATH}" ]; then
+  usage
```

Comment:
Add `exit 1`?

Reply:
`usage()` already has an `exit 1` as the last statement